blocks/cards/cards.css

```diff
@@ -1,3 +1,14 @@
+.section.cards-container.cards-grid {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 990pt;
+  margin: auto;
+}
+
+.section.cards-container.cards-grid .cards-wrapper {
+  flex: 0 0 33.3333%;
```

Comment:
why cards-grid is a section metadata
why can't we have one variant of cards block (i.e grid) and it will be like 3 columns grid or 4 columns grid

Reply:
@jindaliiita I figured section metadata was better because this only occurs once!
Is it worth it to develop another feature if it's hardly used was my reasoning.